### PR TITLE
[TASK] Use Webpacker for SCSS

### PR DESCRIPTION
**Pull request by michsch**

### CODE REVIEW

File: app/packs/entrypoints/application.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+$color-background: #efefef;
```
Comment (oliverklee) [inline app/packs/entrypoints/application.scss:1]: I don't think we need these styles anymore after we've tested that this works. :-)

And do we still need `app/assets/stylesheets/application.css` and `app/assets/stylesheets/welcome.scss`?

File: config/webpack/custom.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    resolve: {
+        extensions: ['.css', '.scss']
```
Comment (oliverklee) [inline config/webpack/custom.js:3]: Do we need to support both file extensions?

File: config/webpack/development.js [hunk 3]
```diff
@@ -2,6 +2,4 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'development'
 
 const webpackConfig = require('./base')
 
-debugger
```
Comment (oliverklee) [inline config/webpack/development.js:5]: :-)

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: app/packs/entrypoints/application.scss (hunk 1 above)
Comment (michsch) [reply app/packs/entrypoints/application.scss:1]: Done

File: config/webpack/custom.js (hunk 2 above)
Comment (michsch) [reply config/webpack/custom.js:3]: Done. custom.js not needed anymore

**Review by oliverklee — APPROVED** [review]

**Pull request merged**